app/views/school_groups/recent_usage.html.erb

```diff
@@ -1 +1,49 @@
 <%= render 'enhanced_header' %>
+
+<table width="100%" class="mt-3 table table-borderless table-sorted school-groups-recent-usage-table">
+  <thead>
+    <tr>
+      <th colspan="2"><%= t('common.electricity') %></th>
+      <th colspan="2"><%= t('common.gas') %></th>
+      <th colspan="2"><%= t('common.storage_heaters') %></th>
```

Comment:
I recognise this wasn't specified anywhere, so happy if you want to do in a follow-up PR, but: very few schools have storage heaters (as far as we know) so the storage heater columns should only appear if at least one school in a group has storage heaters. this avoids having empty columns for (most?) groups.